### PR TITLE
Guide for Flashing LEDs on STM32 using Fedora

### PR DESCRIPTION
**Pull request by rzayevan**

### CODE REVIEW

File: STM32Resources/README.adoc [hunk 1]
```diff
@@ -0,0 +1,66 @@
+= Compiling and Flashing STM Board on Fedora
+
+. Go to UCSolarCarTeam repo > Recruit Resources, and clone.
```
Comment (ExtraGravity) [inline STM32Resources/README.adoc:3]: We should add the resources to our own repo in this PR.

File: STM32Resources/install.sh [hunk 2]
```diff
@@ -6,7 +6,17 @@ if [[ $UID != 0 ]]; then
...
...le.i686 bzip2-libs.i686 cairo.i686 cyrus-sasl-lib.i686 dbus-libs.i686 directfb.i686 esound-libs.i686 fltk.i686 freeglut.i686 gtk2.i686 hal-libs.i686 imlib.i686 lcms-libs.i686 lesstif.i686 libacl.i686 libao.i686 libattr.i686 libcap.i686 libdrm.i686 libexif.i686 libgnomecanvas.i686 libICE.i686 libieee1284.i686 libsigc++20.i686 libSM.i686 libtool-ltdl.i686 libusb.i686 libwmf.i686 libwmf-lite.i686 libX11.i686 libXau.i686 libXaw.i686 libXcomposite.i686 libXdamage.i686 libXdmcp.i686 libXext.i686 libXfixes.i686 libxkbfile.i686 libxml2.i686 libXmu.i686 libXp.i686 libXpm.i686 libXScrnSaver.i686 libxslt.i686 libXt.i686 libXtst.i686 libXv.i686 libXxf86vm.i686 lzo.i686 mesa-libGL.i686 mesa-libGLU.i686 nas-libs.i686 nss_ldap.i686 cdk.i686 openldap.i686 pam.i686 popt.i686 pulseaudio-libs.i686 sane-backends-libs-gphoto2.i686 sane-backends-libs.i686 SDL.i686 svgalib.i686 unixODBC.i686 zlib.i686 compat-expat1.i686 compat-libstdc++-33.i686 openal-soft.i686 alsa-oss-libs.i686 redhat-lsb.i686 alsa-plugins-pulseaudio.i686 alsa-plugins-oss.i686 alsa-lib.i686 nspluginwrapper.i686 libXv.i686 libXScrnSaver.i686 qt.i686 qt-x11.i686 pulseaudio-libs.i686 pulseaudio-libs-glib2.i686 alsa-plugins-pulseaudio.i686
```
Comment (ExtraGravity) [inline STM32Resources/install.sh:11]: This needs to check if it's ubuntu or fedora and install the right ones, not just install fedora ones haha.

File: STM32Resources/install.sh [hunk 3]
```diff
@@ -17,10 +27,11 @@ if ! type "st-flash" > /dev/null; then
     ln -s /opt/stlink/build/st-flash /usr/local/bin/st-flash
     ln -s /opt/stlink/build/st-info /usr/local/bin/st-info
     ln -s /opt/stlink/build/src/gdbserver/st-util /usr/local/bin/st-util
-    echo "export LD_LIBRARY_PATH=\$LD_LIBRARY_PATH:/opt/stlink/build" >> ~/.profile
+    echo "export LD_LIBRARY_PATH=\$LD_LIBRARY_PATH:/opt/stlink/build" >> ~/.bashrc
     export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/opt/stlink/build
-    echo "export PATH=\$PATH:/opt/stlink/build" >> ~/.profile
+    echo "export PATH=\$PATH:/opt/stlink/build" >> ~/.bashrc
```
Comment (ExtraGravity) [inline STM32Resources/install.sh:32]: Same thing here with supporting with fedora and ubuntu.

**Review by ExtraGravity — CHANGES_REQUESTED** [review]
Wow really nice!

File: STM32Resources/README.adoc (hunk 1 above)
Comment (rzayevan) [reply STM32Resources/README.adoc]: Yeah, we didn't have the files. That's why I cloned it from there

File: STM32Resources/install.sh (hunk 2 above)
Comment (rzayevan) [reply STM32Resources/install.sh:11]: Why do we need support for Ubuntu? I thought this guide was just for Fedora

**Pull request closed without merging**